SPECS/xorg-x11-font-utils/xorg-x11-font-utils.spec

```diff
@@ -1,23 +1,21 @@
 %global bdftopcf 1.1
-%global fonttosfnt 1.2.1
-%global mkfontdir 1.0.7
-%global mkfontscale 1.1.3
-%global font_util 1.3.1
+%global fonttosfnt 1.2.3
+%global mkfontscale 1.2.2
+%global font_util 1.4.1
 # Must be kept in sync with xorg-x11-fonts!
 %global _x11fontdir %{_datadir}/X11/fonts
 Summary:        X.Org X11 font utilities
 Name:           xorg-x11-font-utils
 Version:        7.5
-Release:        50%{?dist}
+Release:        51%{?dist}
```

Comment:
Aren't we setting release to 1?

Reply:
the package version itself isn't being updated here therefore the release is bumped

Reply:
Thanks @amritakohli for answering this. Yes @ndubchak , we just upgrade a few packages that xorg-x11-font-utils provides, and xorg-x11-font-utils itself is not upgraded.